public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
Your public folder doesn't have `_redirects` file, it will prevent 404 error on refresh - netlify.

Create a `_redirects` file and add this `/*  /index.html  200`